jamato14/smart-mirror
# Adds cursory datestamp functionality

What does this PR do?

This PR adds a datestamp alongside the clock functionality

How to test?

Currently there are no automated tests. So a manual test is necessary. 
NPM start -> Go to homepage and verify the date and time are correct.
